releases/release-1.22/README.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "Kubernetes 1.22 Release Information"
+weight: 98
+slug: "release"
+aliases: [ "/release" ]
+description: |
+  Information regarding the current release cycle including important dates,
+  Release Team contact information, tracking spreadsheets and more!
+---
+
+# Kubernetes 1.22
+
+#### Links
+
+* [This document](https://git.k8s.io/sig-release/releases/release-1.22/README.md)
+* [Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.22/release-team.md)
+* Meeting Minutes: TODO (join [kubernetes-sig-release@] to receive meeting invites)
+* [v1.22 Release Calendar][k8s122-calendar]
+* Contact: [#sig-release] on slack, [kubernetes-release-team@] on e-mail
+* [Internal Contact Info][Internal Contact Info] (accessible only to members of [kubernetes-release-team@])
+
+#### Tracking docs
+
+* Enhancements Tracking Sheet: TODO
+* Bug Triage Tracking Sheet: TODO
+* CI Signal Report: TODO
+* [Retrospective Document][Retrospective Document]
+* [kubernetes/sig-release v1.22 milestone](https://github.com/kubernetes/kubernetes/milestone/50)
+
+#### Guides
+
+* [Targeting Issues and PRs to This Milestone](https://git.k8s.io/community/contributors/devel/sig-release/release.md)
+* [Triaging and Escalating Test Failures](https://git.k8s.io/community/contributors/devel/sig-testing/testing.md#troubleshooting-a-failure)
+
+## TL;DR
+
+The 1.22 release cycle is proposed as follows:
+
+- **Monday, April 26th**: Week 1 - Release cycle begins
+- **Tuesday, May 25th**: Week 5 - [Enhancements Freeze](../release_phases.md#enhancements-freeze)
+- **Tuesday, July 6th**: Week 11 - [Code Freeze](../release_phases.md#code-freeze)
```

Comment:
Many people will be out for a long-weekend around July 4th... that is likely to make it hard to find reviewers for last minute crunch items. I'm happy for the answer to that to be "don't expect last minute crunch items to make it in" if that's what we want.

Reply:
I'm in favor of "don't expect last minute crunch items to make it in". We should probably make sure we communicate this when we send out comms around the release.

Reply:
+1

Reply:
In the past we've had code freeze before major holidays. I find that helps avoid scenarios where people end up working over the holidays.

Reply:
+1

Reply:
Thinking out loud: before July 4th implies a code freeze the week of June 28th, which is week 10. This would lead to a 5 week freeze/over a month frozen. This seems very long?

Why not do code freeze week 12 and test freeze week 13?

Reply:
If we have the code freeze in week 12, docs deadlines will be pushed. All the 3 docs deadlines need an average of 2.5-3 weeks to be completed and that will be very close to the release.
I am in favor of communicating well in advance about the code freeze and having at least 1 week between the major deadlines and release day.

Reply:
I think it's somewhat ok for the docs deadlines to be pulled forward by a half week or so to make room for a shorter code freeze. See also on the main conversation: https://github.com/kubernetes/sig-release/pull/1530#issuecomment-824288017

Reply:
q: wouldn't code freeze in week 10 be the same as a 4/year release? if the idea is to give people more time and breathing room it seems like adding time there somehow would be welcome/ a value-add for authors/sigs.

Reply:
+1 on pushing the code freeze back to week 12, I would personally appreciate the breathing room.

Reply:
I don't think the docs placeholder deadline should come before the code freeze, especially with how responsive everyone was to the docs team when the deadline came after the code freeze during the last release. If pushing code freeze, I think it might be good to push placeholder to the same day as code freeze or a few days after code freeze.

Reply:
> All the 3 docs deadlines need an average of 2.5-3 weeks to be completed and that will be very close to the release
This is indeed a concern to docs, with a longer release, we are definitely going to have more KEPs to track and if the 1.22 release cycle proves anything, although things went pretty good, is that people leave the docs PR last and are hard to get a hold of. Not only to submit the k/website PRs but also to get someone to do the tech review from their own SIG.
It would be ideal and it would speed the process up if someone commits to doing the technical docs review from the SIG, but i'm not sure if such a process is in place currently.

Reply:
Code freeze has been updated to July 8th and the deadline to open a docs placeholder PR will be July 9th. Test freeze has been updated to July 15 (one week after code freeze)